src/lib/falcor/FalcorController.js

```diff
@@ -103,15 +101,65 @@ export default class FalcorController extends BaseComponent {
         });
       }
       else {
-        const err = new Error("FalcorPathSets: " + JSON.stringify(falcorPathSets) + " returned no data.")
-        console.error(err);
+        console.error("FalcorPathSets: " + JSON.stringify(falcorPathSets) + " returned no data.");
```

Comment:
Maybe demote this to console.warning. This is not necessarily an error, we might just be fetching a category that has no articles for instance.